packages/gensx/src/commands/environment/unselect.ts

```diff
@@ -0,0 +1,52 @@
+import ora from "ora";
+import pc from "picocolors";
+
+import { checkProjectExists } from "../../models/projects.js";
+import { validateAndSelectEnvironment } from "../../utils/env-config.js";
+import { readProjectConfig } from "../../utils/project-config.js";
+
+interface UnselectOptions {
+  project?: string;
+}
+
+export type { UnselectOptions };
+
+export async function handleUnselectEnvironment(options: UnselectOptions) {
```

Comment:
Put this in the docs PR but I don't understand why we need this command.

Reply:
The goal here was to strike the right balance between having the CLI be stateful (remember your selected env) while not having the state surprise users. Essentially balancing the preferences of you and @jmoseley. Without this command, a user could get locked into having the CLI be stateful. 

Also worth noting that pulumi offers the same command for their stack CLI: https://www.pulumi.com/docs/iac/cli/commands/pulumi_stack/

Reply:
huh, TIL